images/virtualization-artifact/pkg/controller/importer/importer_pod.go

```diff
@@ -62,6 +62,15 @@ const (
 type Importer struct {
 	PodSettings *PodSettings
 	EnvSettings *Settings
+	pvcName     string
+}
+
+func NewImporterFromPVC(podSettings *PodSettings, envSettings *Settings, pvcName string) *Importer {
+	return &Importer{
+		PodSettings: podSettings,
+		EnvSettings: envSettings,
+		pvcName:     pvcName,
+	}
```

Comment:
I think this method is a quick hack. We should add PVC name to the PodSettings, and add new clause with volume device to the addVolumes method.

Reply:
Fixed.